src/gluonts/model/wavenet/_estimator.py

```diff
@@ -175,7 +175,7 @@ def __init__(
         seasonality: Optional[int] = None,
         embedding_dimension: int = 5,
         num_bins: int = 1024,
-        bin_centers: Optional[List[float]] = None,
+        bin_centers: Optional[np.ndarray] = None,
```

Comment:
This could be more general: `Optional[Collection[float]]` and then you cast it to `np.array` further below